apps/staking/app/mystakes/modules/claim/ClaimTokens.tsx

```diff
@@ -141,7 +147,15 @@ export function ClaimTokens({
             },
           ]}
           hasMissingEstimatesTooltipContent={dictionaryFee('missingFees')}
-          gasHighTooltip={dictionaryFee.rich('gasHigh', { link: externalLink(URL.GAS_INFO) })}
+          gasHighTooltip={
+            <WizardSectionDescription
+              className="text-base md:text-base"
+              description={dictionaryFee.rich('gasHigh', {
+                linkOut: '',
+              })}
+              href={URL.GAS_INFO}
+            />
+          }
```

Comment:
TODOLATER this gas wizard section occurs a lot it might be worth making them components in the future